src/main/java/ru/l3r8y/parser/ClassMethods.java

```diff
@@ -107,10 +104,7 @@ private void processNodeIfClassDeclaration(final Collection<Method> methods, fin
             if (
                 !new IsSuppressed(
                     suppressed,
-                    new ListOf<>(
-                        "MutableStateCheck",
-                        "ErSuffixCheck"
-                    )
+                    new ClassCanSuppress().value()
```

Comment:
@h1alexbel also I don't think this is the best practice for solving this problem.

The problem is that you're creating a scalar here and immediately calling `#value`, which is procedural code - just like calling a static method. You're kind of doing an inline

Reply:
you are right, its a not a proper way

Reply:
@l3r8yJ got fixed in the latest commit